apps/modernization-ui/src/pages/patient/profile/addresses/AddressesTable.tsx

```diff
@@ -79,6 +81,8 @@ type Props = {
 
 export const AddressesTable = ({ patient }: Props) => {
     const { showAlert } = useAlert();
+    const { id } = useParams();
```

Comment:
Can a property be added to the table to like the `EditableCard`?  The actual identifier of the patient is already passed as the `patient` property but that' can't be used with the `usePatientProfile` hook because it expects the short-id.  Grabbing the short id from the params is placing a dependency on the URL.

Reply:
So the EditableCard always has an Edit button on its top right. 
But that's not the case for a Table component. You don't always have an Edit button in a table component level.